_maps/map_files/cyberiad/z3.dmm

```diff
@@ -1,357 +1,234 @@
+"dv" = (/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/machinery/light{dir = 8},/turf/simulated/floor/plating,/area/tcommsat/powercontrol)
+"dw" = (/obj/effect/decal/warning_stripes{icon_state = "2"},/turf/simulated/floor/plating,/area/tcommsat/powercontrol)
+"dx" = (/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/machinery/atmospherics/pipe/manifold/hidden/supply{dir = 8; initialize_directions = 11; level = 1},/turf/simulated/floor/plasteel{icon_state = "dark"},/area/turret_protected/tcomfoyer)
+"dy" = (/obj/effect/decal/warning_stripes{icon_state = "SW-out"},/turf/simulated/floor/plasteel{icon_state = "darkred"; dir = 8},/area/turret_protected/tcomfoyer)
+"dz" = (/obj/machinery/atmospherics/unary/vent_pump{dir = 8; on = 1},/obj/machinery/light{dir = 4},/turf/simulated/floor/plasteel{icon_state = "darkred"; dir = 4},/area/turret_protected/tcomfoyer)
+"dA" = (/obj/effect/spawner/window/reinforced,/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"; tag = ""},/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"; tag = ""},/obj/structure/cable{d2 = 2; icon_state = "0-2"},/turf/simulated/floor/plating,/area/tcommsat/entrance)
+"dB" = (/obj/effect/spawner/window/reinforced,/obj/structure/cable{d2 = 8; icon_state = "0-8"},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0},/turf/simulated/floor/plating,/area/tcommsat/entrance)
+"dC" = (/obj/machinery/door/airlock/external{frequency = 1381; icon_state = "door_locked"; id_tag = "telecoms_outer"; locked = 1; name = "External Access"; req_access = null; req_access_txt = "10;13"},/turf/simulated/floor/plating,/area/tcommsat/powercontrol)
+"dD" = (/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/item/radio/intercom{dir = 0; name = "station intercom (General)"; pixel_x = -28},/turf/simulated/floor/plating/airless,/area/tcommsat/powercontrol)
+"dE" = (/obj/machinery/atmospherics/pipe/manifold/visible{dir = 1},/turf/simulated/floor/plating,/area/tcommsat/powercontrol)
+"dF" = (/obj/machinery/atmospherics/pipe/simple/visible{dir = 6; level = 1},/turf/simulated/floor/plating,/area/tcommsat/powercontrol)
+"dG" = (/obj/machinery/door/airlock/maintenance_hatch{name = "Telecomms Power/Atmospherics"; req_access_txt = "61"},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0},/obj/machinery/atmospherics/pipe/simple/hidden/supply{dir = 4; level = 1},/turf/simulated/floor/plasteel{icon_state = "dark"},/area/tcommsat/powercontrol)
+"dH" = (/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"; tag = ""},/obj/machinery/atmospherics/pipe/simple/visible/universal{dir = 4},/obj/effect/decal/warning_stripes{icon_state = "E-corner"},/turf/simulated/floor/plating,/area/tcommsat/powercontrol)
+"dI" = (/obj/structure/cable{d1 = 1; d2 = 8; icon_state = "1-8"},/obj/struct
```

Comment:
Why did you place so many exterior lights? There is literally zero reason for them to be there, for space will light those areas properly on its own anyways. And even if it did not, it's still an overkill.

Reply:
I dont know the spread on lights, and they are there for maintenance on the solars

Reply:
Well, I would say none of the exterior lights are needed. You will see without them just as fine because of the space light.

Reply:
Gone